src/AppBundle/Action/DefaultAction.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace AppBundle\Action;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\ControllerTrait;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\HttpFoundation\Request;
+
+class DefaultAction
+{
+    use ControllerTrait;
+
+    private $container;
+
+    public function __construct(ContainerInterface $container)
```

Comment:
As discussed during the hackday, it would be better to inject the parameter directly but it requires to add some lines in `services.yml`. IMO it's very rare that a controller should use a parameter (parameters must usually be injected in standard services). Maybe the best for here is to remove the use of this parameter?